paprika/analysis.py

```diff
@@ -289,18 +290,18 @@ def results(self):
 
         The final free energy and uncertainty estimate is specified in the ``fe`` and ``sem`` keys.
 
-        If multiple fractions of the data are specified, the free energy and SEM from each fraction are stored in a nested
-        dictionary under the keys ``fraction_fe`` and ``fractrion_sem``. The number of frames analyzed for each fraciton
-        is stored under ``fraction_n_frames``.
+        If multiple fractions of the data are specified, the free energy and SEM from each fraction are stored in
```

Comment:
@jeff231li "fraciton" and "fractrion_sem" still remain as misspelled "fraction" in these three lines

Reply:
oops, thanks for spotting this.